yt_dlp/extractor/mediasite.py

```diff
@@ -167,11 +167,6 @@ def __extract_slides(self, *, stream_id, snum, Stream, duration, images, title,
             'format_note': 'Slides',
             'fragments': fragments,
             'fragment_base_url': slide_base_url,
-            '_download_params': {
-                'duration': duration,
```

Comment:
Now that this is removed, how is duration obtained in the downloader?

Reply:
I did not see this `duration` being used anywhere and thus assumed it is left-over from an earlier version of the code.

If needed, the video duration can be obtained directly from `infodict.get('duration')`, but is it needed anywhere?

Reply:
Oh, right. I thought the slide durations were computed in the downloader from time codes, but then I remembered I compute those in the Mediasite extractor instead. My initial draft may have done the former, and as such needed to know the duration of the whole video.

So it really isn't needed any more.